docs/topic-guides/blendable-web-apps/view-model-attaching.md

```diff
@@ -108,5 +108,5 @@ Here is the explanation of each field:
 * `Token`: Attachment rule token described above. Default value is an empty string which is treated as "DefaultToken".
 * `Active`: Defines default attachment rule rule activeness. Default value is "True".
 * `Contexts`: Attachment rule contexts (if any). Default value is "null".
-* `AllowFromDirection`: Allows attachment rule calls from this URI (it defines if this URI can trigger attachment process). Default value is "True".
-* `AllowToDirection`: Allows attachment rule calls to this URI (it defines if this URI can be triggered during attachment process). Default value is "True".
+* `AllowFromDirection`: Allows attachment rule calls from this URI (it defines if this URI can trigger attachment process). That means if It's set to true, when an app calls `Self.GET()` on an attachment rule URI it will request other apps for content with the same contexts, otherwise it will not. Default value is "True".
+* `AllowToDirection`: Allows an attachment rule calls to this URI (it defines if this URI can be triggered during attachment process). That means if It's set to true, when other app requests on attachment rule with the same contexts as one of the attachment rules described in your app, your app will respond, otherwise it will not. Default value is "True".
```

Comment:
Remove the "an" before "attachment rule calls" and use lowercase "it's" and you're good to go! The rephrasing makes it much clearer

Reply:
Thank you!